app/controllers/api/storage.php

```diff
@@ -800,7 +800,8 @@
 App::get('/v1/storage/buckets/:bucketId/files/:fileId/preview')
     ->alias('/v1/storage/files/:fileId/preview', ['bucketId' => 'default'])
     ->desc('Get File Preview')
-    ->groups(['api', 'storage'])
+    ->groups(['api', 'storage', 'cache'])
+    ->label('useCache', true)
```

Comment:
We usually use a namespace convention for labels.
```suggestion
    ->label('cache', true)
```

Reply:
changed